src/deployments.ts

```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V0,
+  startBlock: 28399140,
 } as const;
 
 export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x4e52b73aa28b7ff84d88ea3a90c0668f46043450',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V1,
+  startBlock: 36825974,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_FUJI_TESTNET_V0 = {
   contractAddress: '0x42816FA3cB0aDc3fcAdED3109323c0Bc19215084',
   network: NETWORK_AVALANCHE_FUJI_TESTNET,
   contractType: RenftContractType.WHOOPI,
   version: WhoopiVersion.V0,
+  startBlock: 11500156,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_MAINNET_V0 = {
   contractAddress: '0x6Ee495ecEd3A0255057667FF2685e53f54A19A65',
   network: NETWORK_AVALANCHE_MAINNET,
   contractType: RenftContractType.WHOOPI,
   version: WhoopiVersion.V0,
+  startBlock: 19408332,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_ETHEREUM_MAINNET_V0 = {
   contractAddress: '0x945e589a4715d1915e6fe14f08e4887bc4019341',
   network: NETWORK_ETHEREUM_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 12875506,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_POLYGON_MAINNET_V0 = {
   contractAddress: '0x6884d88Ce56C5C93F46eE23684eBA8628c90B518',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 28399112,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x3ddc85bb768a11b0125f4ee71cfea54e54653366',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V1,
+  startBlock: 36825213,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_AVALANCHE_FUJI_TESTNET_V0 = {
   contractAddress: '0x23F7F8B03BAF01D5124255fE240E81BbBd3AEc0D',
   network: NETWORK_AVALANCHE_FUJI_TESTNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 11500156,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_AVALANCHE_MAINNET_V0 = {
   contractAddress: '0xEBFd584AAC21dfEFF02c3d4f308B0962610a028A',
   network: NETWORK_AVALANCHE_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 19408332,
```

Comment:
Struggling to find these values: https://snowtrace.io/address/0xEBFd584AAC21dfEFF02c3d4f308B0962610a028A

Reply:
contract creation txn: https://snowtrace.io/tx/0x5c913c9d83b72784199b26e2c46c1cad184960df1c1f74754d747fc32b9aa129